packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -38,6 +38,7 @@ import {
 } from "@calcom/features/ee/workflows/lib/allowDisablingStandardEmails";
 import { scheduleWorkflowReminders } from "@calcom/features/ee/workflows/lib/reminders/reminderScheduler";
 import { getFullName } from "@calcom/features/form-builder/utils";
+import { sendNotification } from "@calcom/features/notifications/sendNotification";
```

Comment:
where are you calling this `sendNotification` function?

Reply:
Currently it's not being used anywhere. I was using this `sendNotification` when a new meeting was booked with requires confirmation setting to send notifications for unconfirmed meetings. But there were some refactors in the codebase and then I couldn't find the code for that.

Also, I thought that I would confirm with the team if there is any central place from which all emails are sent so the browser notifications can also be sent from there for all events.